db/data/20211228070332_remove_sharp_and_octothorp_of_tag_name_head.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RemoveSharpAndOctothorpOfTagNameHead < ActiveRecord::Migration[6.1]
+  BEGINNING_WITH_SHARP_OR_OCTOTHORP = '^(#|＃|♯)( |　)*.*'
+
+  def up
+    ActsAsTaggableOn::Tag.where('name ~* ?', BEGINNING_WITH_SHARP_OR_OCTOTHORP).find_each do |tag_to_remove_name_head|
+      head_removed_name = tag_to_remove_name_head.name.sub(/^(#|＃|♯)( |　)*/, '')
+      if ActsAsTaggableOn::Tag.exists?(name: head_removed_name)
+        existing_tag_id = ActsAsTaggableOn::Tag.find_by(name: head_removed_name).id
```

Comment:
[pick](https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pick) を使うと少し短く書けそうです。
SQLの回数も（元コードは exists と select の2回なので）1回で済むようになる。

```suggestion
      existing_tag_id = ActsAsTaggableOn::Tag.where(name: head_removed_name).pick(:id)
      if existing_tag_id
```

Reply:
@sinsoku 
引き続いてコードをご確認くださり、本当にありがとうございます！
勉強になりました。このようにさせて頂きます🙏
（GitHub でこのように変更を提案する機能があることも同時に勉強させて頂きました…！）